test/test-utils/test-utils.ts

```diff
@@ -219,6 +221,15 @@ export function mkEvent(opts: MakeEventProps): MatrixEvent {
         event.state_key = "";
     }
 
+    // @ts-ignore
+    event.getRoomId = () => opts.room;
+    // @ts-ignore
+    event.getType = () => opts.type;
+    // @ts-ignore
+    event.getStateKey = () => opts.skey;
+    // @ts-ignore
+    event.getContent = () => opts.content;
```

Comment:
Shouldn't these be set on the `mxEvent` rather than the raw `event`?

Reply:
They shouldn't be set at all :laughing: Just found out what the `event` param does :neutral_face: